### ISSUE TITLE
Suggestion: Box Plots

### ISSUE THREAD

**Issue by jsenon** [fortio/fortio#230]
Is it possible to add graph overview with Box Plots in addition to bar chart?
https://github.com/gonum/plot/wiki/Example-plots#box-plots

**Comment by ldemailly** [comment]
found https://github.com/chartjs/Chart.js/issues/4275 which points to 2 external implementations; asked about possible "built in" support

**Comment by marclaporte** [comment]
I agree with @ldemailly  It would be much better to have one good implementation built-in and everyone converging efforts.

External plugins cause no end of pain, as explained here:  http://pluginproblems.com/
